test_util/launch_configs/README.md

```diff
@@ -0,0 +1,50 @@
+# DC/OS Launch Configuration YAML
+## Design Intention
+The intention of this configuration file is to provide an interface by which
+all deployments of DC/OS, regardless of provider have a similar format, thus
+complementing the goal of dcos-launch to provide a single tool for launching
+across a variety of provider APIs
+
+## Supported Deployments and Examples
+- [Simple AWS Cloudformation](aws-cf.yaml)
+- [Zen AWS Cloudformation](aws-zen-cf.yaml)
+- [Non-DC/OS AWS Linux Cluster](aws-linux-cluster.yaml)
```

Comment:
This is what is currently called a "VPC" by CCM. I.E. a bunch of hosts with some standard distro loaded that has been provided by AWS